.github/workflows/main.yml

```diff
@@ -47,9 +47,15 @@ jobs:
         run: poetry install
         if: steps.cache.outputs.cache-hit != 'true'
 
-      # step by step
-      # - name: Code Quality Black
-      #   run: poetry run black . --check
+      # Code quality
+      - name: Code Quality Black
+        run: poetry run black . --check
+
+      - name: Code quality Flake8
+        run: poetry run flake8 . --ignore=E203,E266,E501,W503,W504,E741 --exclude=.git,__pycache__,.venv,.downloads,fhirzeug/generators/python/,fhirzeug/generators/python_pydantic/templates/
```

Comment:
@julian-r : I can see that putting flake8 configuration inside the github workflow file is ugly but I did not find a better way of doing it yet. Flake8 project doesn't seem to support `pyproject.toml` config file and debate is raging on the subject : https://gitlab.com/pycqa/flake8/-/issues/428 . I did not want to include a `setup.cfg` file only for flake8 support so I left it here. 
Do you have any better solution to suggest ?

Reply:
for me a [`.flake8`](https://flake8.pycqa.org/en/2.5.5/config.html#per-project) file would be fine.